best-practices.md

```diff
@@ -202,12 +288,72 @@ register the media type with IANA, so that other STAC clients can find it easily
 [acceptable to not register](https://stackoverflow.com/questions/29121241/custom-content-type-is-registering-with-iana-mandatory) 
 it. It is relatively easy to [register](https://www.iana.org/form/media-types) a `vnd` media type.
 
-## Static and Dynamic Catalogs
+### Asset Roles
+
+[Asset roles](item-spec/item-spec.md#asset-roles) are used to describe what each asset is used for. They are particular useful 
+when several assets have the same media type, such as when an Item has a multispectral analytic asset, a 3-band full resolution 
+visual asset, a downsampled overview, and an cloud mask, all stored as Cloud Optimized GeoTIFF's. It is recommended to use a role
+for every asset available, and just pick a sensible name for the role if there is not a clear one in the [Asset Role 
+Types](item-spec/item-spec.md#asset-role-types). You can also propose new roles to the specification by raising an issue/PR. 
+
+In addition to the thumbnail, data and overview [roles listed](item-spec/item-spec.md#asset-role-types) in the Item spec, there
+are a number of roles that are emerging in practice, but don't have enough widespread use to justify standardizing them. So if
+you want to re-use other roles then try to find them on the list below, and also feel free to suggest more to include here.
+
+#### List of Asset Roles
+
+| Role Name | Description                                                                           |
+| --------- | ------------------------------------------------------------------------------------- |
+| overview  | An asset that represents a possibly larger view than the thumbnail of the Item, for example, a true color composite of multi-band data. |
+| visual    | An asset that is a full resolution version of the data, processed for visual use (RGB only, often sharpened ([pan-sharpened](https://en.wikipedia.org/wiki/Pansharpened_image) and/or using an [unsharp mask](https://en.wikipedia.org/wiki/Unsharp_masking))). |
+| reflectance | An asset the provides [reflectance](https://www.l3harrisgeospatial.com/Support/Self-Help-Tools/Help-Articles/Help-Articles-Detail/ArtMID/10220/ArticleID/19247/3377) values, instead of just radiance. |
+| temperature | An asset that provides actual temperature measurements. |
+| date | An asset that provides per-pixel acquisition timestamps, typically serving as metadata to another asset |
+| saturation | Points to a file that indicates where pixels in the input spectral bands are saturated. |
+| cloud | Points to a file that indicates whether a pixel is assessed as being cloud |
+| cloud-shadow | Points to a file that indicates whether a pixel is assessed as being cloud shadow. |
```

Comment:
do we have the convention of using kebab-case for enums described anywhere in the spec?

Reply:
No, I made this decision arbitrarily in the CARD4L extensions (where this list originates from).

Having that said, I'm not sure we should list these all here, but limit to things that seem generally used like cloud-cover, cloud-shadow, snow/ice, land/water, ... 
The issue with this list is that is only captures a small subset of use cases and could become **very* long or always incomplete. Some things are also very specific like date, date-offset, noise-power, gamma-sigma... backscatter, prd, covmat are specific products, but by no mean complete.

Reply:
Cool. I'm up for a more curated list. But I do want us to list a number of potentially relevant ones - I just didn't have enough knowledge of all the options to cut it down. I do think it's ok if it's a pretty long list, and if it's incomplete, just so people don't end up making their own things that are all slightly different.